drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -166,6 +166,8 @@ static int init_hfxo(const struct device *dev)
 		return rc;
 	}
 
+	dev_data->start_up_time = K_USEC(NRF_APPLICATION_BICR_NS->HFXO.STARTUPTIME);
```

Comment:
`NRF_APPLICATION_BICR_NS` is defined only for the Application core, so this will not compile for other cores. I guess we'll need to use some hack instead of this symbol until MDK is updated.
Also, I think an error should be returned here if this value turns out to be unconfigured.

Reply:
My idea was to define BICR in DT, so we'd obtain it from there. I guess it is accessible from other cores, like radio @hubertmis ?

Good point about handling uninitialized value.

Reply:
Yes, UICR (including BICR) is readable by all CPUs